DROD/GameScreen.cpp

```diff
@@ -1553,6 +1553,10 @@ void CGameScreen::OnKeyDown(
 	}
 	if (nCommand != CMD_UNSPECIFIED)
 	{
+		if (nCommand == CMD_WAIT && bMacro && Key.repeat == 1) {
```

Comment:
Suggest adding a comment to explain how this logic helps, otherwise this looks unintuitive.